src/app/checkout/step-3/step-3.component.js

```diff
@@ -190,6 +190,7 @@ export default angular
       onSubmitted: '&',
       onSubmittingOrder: '&',
       submittingOrder: '<',
+      premiumName: '<',
```

Comment:
I don't think we want to display premiums on standard checkout for now. So no changes to `step-3`.

Reply:
`branded-checkout-step-2` is basically just a wrapper around `checkout/step-3`.  I want to show the premium for branded checkout, but not for standard checkout.  So I added conditional logic to `step-3` so that it would render the premium section only if we pass `premiumName` to it.  Would that be fine?  Otherwise, I'm not sure where else it make sense to display the premium in https://github.com/CruGlobal/give-web/blob/master/src/app/branded/step-2/branded-checkout-step-2.tpl.html

Reply:
Ah, I see. The one thing that would need to be tested with that is switching between branded and standard checkout. If you're storing premium name in local storage, it will still be populated when going to standard checkout, right? Though, given my other comments in this PR, you might be able to get rid of it from local storage. Even so, it would be stored on the gift and we'd have the same issue. At that point we have 2 options: show it on standard checkout in the rare cases it is still in there, but do it nicely; or put some sort of variable in to know whether or not it is branded checkout and only show it there.

Reply:
Yeah I'm thinking that we don't want to display it or submit it if it's not branded checkout.  One idea I have is to handle it similar to `checkout/step-2`: step 2 has a `brandedCheckoutItem` property that essentially kills two birds with one stone.  It lets step 2 know that it's branded checkout, and it contains the relevant data needed for branded checkout.  I could do something similar here.  It just so happens that `brandedCheckoutItem` in this context is set to `itemConfig`, which seems to be the place I should be storing `premiumCode`.

Reply:
https://github.com/CruGlobal/give-web/blob/master/src/app/branded/step-1/branded-checkout-step-1.tpl.html#L37

Reply:
I'm ok with you using that.